tips/025_tip_create_new_authority.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: How to create a new Authority
+sitemap:
+priority: 0.1
+lastmod: 2018-10-05T18:20:00-00:00
+---
+# How to create a new Authority
+
+Tip submitted by @Tonterias
+
+Let's say that you need a new authority besides the given ones of ADMIN and USER.
+
+Modify /src/main/java/com/jhipsterpress/web/security/AuthoritiesConstants.java file to include your new authorities:
+
+	/**
+	 * Constants for Spring Security authorities.
+	 */
+	public final class AuthoritiesConstants {
+	
+	    public static final String ADMIN = "ROLE_ADMIN";
+	
+	    public static final String USER = "ROLE_USER";
+	
+	    public static final String ANONYMOUS = "ROLE_ANONYMOUS";
+	
+	    private AuthoritiesConstants() {
+	    }
+	}
+
+Do not forget to include your new role in your /src/main/resources/config/liquibase/authorities.csv:
+
+	name
+	ROLE_ADMIN
+	ROLE_USER
+	ROLE_ANONYMOUS
+
+
+With that, you will be able to use it in your /src/main/java/es3/config/SecurityConfiguration.java or in (src/main/java/com/jhipsterpress/web/web/rest/FrontpageconfigResource.java), for example:
```

Comment:
Same thing for the package name here, and `FrontpageconfigResource` is probably something specific for your project?

Reply:
Ok, I explain it for newbies lke me, so I tend to be super detailed. Should I change it in a new Pull Request or have you changed it? This is my second PR. Thanks for your help.